tests/integration/kubernetes/gha-run.sh

```diff
@@ -33,6 +33,7 @@ HTTPS_PROXY="${HTTPS_PROXY:-${https_proxy:-}}"
 NO_PROXY="${NO_PROXY:-${no_proxy:-}}"
 export AUTO_GENERATE_POLICY="${AUTO_GENERATE_POLICY:-no}"
```

Comment:
The workflow YAML uses `true/false` instead of `yes/no` so let's be consistent and default this to `false`